lib/src/analytics.dart

```diff
@@ -44,12 +44,12 @@ class Analytics extends JsObjectWrapper<analytics_interop.AnalyticsJsImpl> {
     }
   }
 
-  void setUserProperties(CustomParams properties,
+  void setUserProperties(Map<String, String> properties,
```

Comment:
Please delete the `CustomParams` and `CustomParamsJSImpl` classes!

Reply:
thanks for the review, it is done